secure_config.py

```
@@ -0,0 +1,220 @@
+        raise RuntimeError(
+            "cryptography library not installed - encryption is required for secure storage. "
+            "Install with: pip install cryptography"
+        )
+
+    try:
+        key = get_or_create_encryption_key()
+        fernet = Fernet(key)
+        encrypted = fernet.encrypt(value.encode())
+        return f"encrypted:{encrypted.decode()}"
+    except Exception as e:
+        logger.error("Encryption failed: %s", e)
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)